### PR TITLE
Add linux script to generate disk image

### PR DESCRIPTION
**Pull request by LanceLi1416**
Script to generate the disk image files.

```sh
generate-disk.bash [-s size] [-f filesystem] [-h help]
  -s, --size        Set the size (e.g., 500M, 10G)
  -f, --file-system Set the file system (fat16, fat32, exfat, ext2, ext4)
  -h, --help        Display this help message
 ```

> Note: this script will generate the disk image in the `KidneyOS` root directory instead of the `scripts` sub-drectory.

e.g.,
```bash
scripts/generate-disk.bash -s 50MiB -f fat16
```

### CODE REVIEW

**Review by 1whatleytay — APPROVED** [review]
Wait till someone else approves this before merging 👍

File: Makefile [hunk 1]
```diff
@@ -93,11 +99,11 @@ run-bochs: $(ISO)
 
 # QEMU_FLAGS := -no-reboot -no-shutdown -m 4G -d int,mmu,pcall,cpu_reset,guest_errors -cdrom $(ISO)
 QEMU_FLAGS := -no-reboot -no-shutdown -m 4G -d int,mmu,pcall,cpu_reset,guest_errors -cdrom $(ISO) \
-			  -drive format=raw,file=gpt_fat16_50mb.img,if=ide \
+			  -drive format=raw,file=${ATADISK},if=ide \
 			  -boot d
 
 .PHONY: run-qemu
-run-qemu: $(ISO)
+run-qemu: $(ISO) $(ATADISK)
```
Comment (1whatleytay) [inline Makefile:106]: Oh and before merging don't generate the disk by default.

Comment (LanceLi1416) [reply Makefile]: Updated!

File: scripts/generate-disk.bash [hunk 2]
```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
+
+# Functions ----------------------------------------------------------------------------------------
+
+# Usage / help
+usage() {
+  # echo "Usage: $0 [-s size] [-p partition] [-f filesystem] [-h help]"
```
Comment (ff-zhang) [inline scripts/generate-disk.bash:7]: NIT: remove if not needed

**Review by ff-zhang — APPROVED** [review]
LGTM

**Pull request merged**